pom.xml

```diff
@@ -38,21 +38,21 @@
     <python.ver>python3 &gt;= 3.0</python.ver>
     <deb.python.ver>python3 (&gt;= 3.0)</deb.python.ver>
     <slf4j.version>2.0.0</slf4j.version>
-    <zookeeper.version>3.5.9</zookeeper.version>
-    <kafka.version>2.8.1</kafka.version>
-    <phoenix.version>5.1.2</phoenix.version>
-    <hbase.version>2.4.13</hbase.version>
+    <zookeeper.version>3.7.2</zookeeper.version>
+    <kafka.version>2.8.2</kafka.version>
+    <phoenix.version>5.1.3</phoenix.version>
+    <hbase.version>2.4.17</hbase.version>
     <curator.version>4.2.0</curator.version>
     <commons-lang3.version>3.12.0</commons-lang3.version>
-    <hbase.tar>http://repo.bigtop.apache.org.s3.amazonaws.com/bigtop-stack-binary/3.2.0/centos-7/x86_64/hbase-2.4.13-bin.tar.gz</hbase.tar>
-    <hbase.folder>hbase-2.4.13</hbase.folder>
-    <hadoop.tar>http://repo.bigtop.apache.org.s3.amazonaws.com/bigtop-stack-binary/3.2.0/centos-7/x86_64/hadoop-3.3.4.tar.gz</hadoop.tar>
-    <hadoop.folder>hadoop-3.3.4</hadoop.folder>
-    <hadoop.version>3.3.4</hadoop.version>
+    <hbase.tar>https://archive.apache.org/dist/hbase/2.4.17/hbase-2.4.17-bin.tar.gz</hbase.tar>
+    <hbase.folder>hbase-2.4.17</hbase.folder>
+    <hadoop.tar>https://archive.apache.org/dist/hadoop/common/hadoop-3.3.6/hadoop-3.3.6.tar.gz</hadoop.tar>
+    <hadoop.folder>hadoop-3.3.6</hadoop.folder>
+    <hadoop.version>3.3.6</hadoop.version>
```

Comment:
should we have the hadoop version in sync with ambari where it is 3.3.4 currently ?
or bump up the version in ambari as well ?

Reply:
Nice one, I think we should bump in ambari also.